ormconfig.js

```diff
@@ -1,17 +1,42 @@
-const baseFolder = process.env.NODE_ENV === "production" ? "build" : "src";
+const _ = require('lodash');
 
-module.exports = {
-  type: "postgres",
-  host: process.env.POSTGRESQL_HOST || "localhost",
-  port: parseInt(process.env.POSTGRESQL_PORT || "5432"),
-  username: process.env.POSTGRESQL_USER,
-  password: process.env.POSTGRESQL_PASSWORD,
-  database: process.env.POSTGRESQL_DB || "ethereum_backend_dev",
+const baseFolder = process.env.NODE_ENV === 'production' ? 'build' : 'src';
+
+const databaseURL = process.env.DATABASE_URL;
+
+const opts = {
+  type: 'postgres',
   entities: [__dirname + `/${baseFolder}/db/entities/*.{ts,js}`],
-  synchronize: process.env.NODE_ENV === "test",
+  synchronize: process.env.NODE_ENV === 'test',
   logging: false,
-  migrations: [`${baseFolder}/migrations/*.ts`],
+  migrations: [`${baseFolder}/migrations/*.{ts,js}`],
   cli: {
     migrationsDir: `${baseFolder}/migrations`,
   },
 };
+
+if (databaseURL) {
+  module.exports = _.extend(
+    {
+      url: databaseURL || 'localhost',
+      ssl: true,
+      extra: {
+        ssl: {
+          rejectUnauthorized: false,
+        },
+      },
+    },
+    opts,
+  );
+} else {
+  module.exports = _.extend(
+    {
+      host: process.env.POSTGRESQL_HOST || 'localhost',
+      port: parseInt(process.env.POSTGRESQL_PORT || '5432'),
+      username: process.env.POSTGRESQL_USER,
+      password: process.env.POSTGRESQL_PASSWORD,
+      database: process.env.POSTGRESQL_DB || 'ethereum_backend_dev',
+    },
+    opts,
+  );
+}
```

Comment:
If databaseURL is the default scenario in production I would use happy path and write !databaseURL as the exception

I haven't tried this because dont have a databaseURL I can test against but I think this would work and avoids the else statement, then the desired situation is at the bottom with the exception above it.

```suggestion
if (!databaseURL) {
  module.exports = _.extend(
    {
      host: process.env.POSTGRESQL_HOST || 'localhost',
      port: parseInt(process.env.POSTGRESQL_PORT || '5432'),
      username: process.env.POSTGRESQL_USER,
      password: process.env.POSTGRESQL_PASSWORD,
      database: process.env.POSTGRESQL_DB || 'ethereum_backend_dev',
    },
    opts,
  );
  
  return;
}

module.exports = _.extend(
  {
    url: databaseURL || 'localhost',
    ssl: true,
    extra: {
      ssl: {
        rejectUnauthorized: false,
      },
    },
  },
  opts,
);
```

Reply:
I'm not sure there's a "default" scenario, but I'll remove the else 👍

Reply:
ahada there's not `return` outside a function xD

Reply:
oh shit :facepalm: 

my bad sorry